src/excmds.ts

```diff
@@ -4304,6 +4306,9 @@ const KILL_STACK: Element[] = []
           - For example, use `bind ;jg hint -Jc .rc > .r > a` on google.com to generate hints only for clickable search results of a given query
         - -br deprecated, use `-qb` instead
         - -F [callback] - run a custom callback on the selected hint, e.g. `hint -JF e => {tri.excmds.tabopen("-b",e.href); e.remove()}`.
+        - -V create hints for invisible elements. By default, elements outside the viewport when calling :hint are not hinted, this includes them anyways.
+        - -! perform action immediately. This effectively selects every hinted element in sequence
```

Comment:
```suggestion
        - -! execute all hints without waiting for a selection
```

I didn't understand what `!` did until I ran it. I think this is a bit better but not perfect

Reply:
Your suggestion looks good to me, at least for now I don't have any other idea.